barretenberg/cpp/src/barretenberg/ultra_honk/ultra_composer.test.cpp

```diff
@@ -207,6 +207,87 @@ TEST_F(UltraHonkComposerTests, create_gates_from_plookup_accumulators)
     prove_and_verify(circuit_builder, /*expected_result=*/true);
 }
 
+/**
+ * @brief Test various failure modes for the lookup relation via bad input polynomials
+ *
+ */
+TEST_F(UltraHonkComposerTests, LookupFailure)
```

Comment:
Not related to this PR, but since we dont have composers anymore maybe these should be called UltraHonk tests

Reply:
also really nice and comprehensive test :)